api/openapi.yaml

```diff
@@ -1600,111 +1776,106 @@ components:
               example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
-            ledgerID:
+            subjectKey:
+              readOnly: true
+              description: |
+                The subject that is granted the entitlement.
               type: string
-              description: The ledger ID.
-              example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
-            void:
+              example: customer-id
+            nextRecurrence:
               description: |
-                If the grant is voided, it will not be applied to the subject's balance anymore.
-              type: boolean
-              readOnly: true
-              example: false
+                The next time the grant will recurr.
+              type: string
+              format: date-time
+              example: "2023-01-01T00:00:00Z"
             expiresAt:
               readOnly: true
               description: |
                 The expiration date of the grant.
               type: string
               format: date-time
               example: "2023-01-01T00:00:00Z"
-    LedgerGrantType:
-      description: |
-        The grant type:
-        - `USAGE` - Increase balance by the amount in the unit of the associated meter.
-      type: string
-      x-enumNames:
-        - LedgerGrantTypeUsage
-      enum:
-        - USAGE
-      example: USAGE
-    LedgerGrantRollover:
-      x-go-type-import:
-        path: github.com/openmeterio/openmeter/internal/credit
-      x-go-type: credit.GrantRollover
+            voidedAt:
+              type: string
+              format: date-time
+              description: The date and time the grant was voided (cannot be used after that).
+              readOnly: true
+    EntitlementValue:
       type: object
-      description: |
-        Grant rollover configuration.
       required:
-        - type
+        - hasAccess
       properties:
-        type:
-          $ref: "#/components/schemas/LedgerGrantRolloverType"
-        maxAmount:
-          description: |
-            Maximum amount to rollover.
+        hasAccess:
+          description: Whether the subject has access to the feature.
+          readOnly: true
+          type: boolean
+          example: true
+        balance:
+          description: The balance of a metered entitlement.
+          readOnly: true
           type: number
-    LedgerGrantRolloverType:
-      x-go-type-import:
-        path: github.com/openmeterio/openmeter/internal/credit
-      x-go-type: credit.GrantRolloverType
-      description: |
-        The rollover type to use:
-        - `REMAINING_AMOUNT` - Rollover remaining amount.
-        - `ORIGINAL_AMOUNT` - Rollover re-applies the full grant amount.
-      type: string
-      enum:
-        - REMAINING_AMOUNT
-        - ORIGINAL_AMOUNT
-      example: ORIGINAL_AMOUNT
-    LedgerGrantExpirationPeriod:
+          format: double
+          example: 100
+        usage:
+          description: Total usage of the feature in the period. Includes overages.
```

Comment:
> The total usage of a metered entitlement for the entitlement period.
> This is the full usage, including overage not covered by grants.

Reply:
will rewrite api descriptions as part of documentation effort